sql/src/lib.rs

```diff
@@ -160,7 +160,14 @@ impl QueryBuilder for SQLiteQueryBuilder {
             &Ref(entid) => self.push_sql(entid.to_string().as_str()),
             &Boolean(v) => self.push_sql(if v { "1" } else { "0" }),
             &Long(v) => self.push_sql(v.to_string().as_str()),
-            &Double(OrderedFloat(v)) => self.push_sql(v.to_string().as_str()),
+            &Double(OrderedFloat(v)) => {
+                // Rust's floats print without a trailing '.' in some cases.
```

Comment:
TIL.  Is there a potential loss of resolution?  That is, I'm wondering if `a.bcdefEx` and `abcdef.0` might not represent different IEEE floats?  I believe the resolution for very large and very small numbers is different, and "naive" parsing changes very large to very small (with high resolution) floats, but I have really not thought this through.

Would https://github.com/rust-lang/rust/issues/30967#issuecomment-172315537 (`{:.1}`) address the loss of resolution?